diff --git a/lib/ymodels.d.ts b/lib/ymodels.d.ts
index 95ed330..10f65b5 100644
--- a/lib/ymodels.d.ts
+++ b/lib/ymodels.d.ts
@@ -14,7 +14,7 @@ export interface IYText extends models.ISharedText {
     readonly undoManager: Y.UndoManager | null;
 }
 export declare type YCellType = YRawCell | YCodeCell | YMarkdownCell;
-export declare class YDocument<T> implements models.ISharedDocument {
+export declare class YDocument<T extends models.DocumentChange> implements models.ISharedDocument {
     get dirty(): boolean;
     set dirty(value: boolean);
     /**
